.gitignore

```diff
@@ -68,4 +68,5 @@ fabric.properties
 gradle
 gradlew
 gradlew.bat
-.gradle
\ No newline at end of file
+.gradlei
```

Comment:
Was this intentional? It's fine if so, wasn't aware Gradle had renamed their settings directory